js/models/listing/Listing.js

```diff
@@ -253,6 +256,8 @@ export default class extends BaseModel {
     const parsedResponse = response.listing;
 
     if (parsedResponse) {
+      // set the hash
+      parsedResponse.listingHash = response.hash;
```

Comment:
That, and it's called listingHash in the order.item, which is the only thing it's used for here.